wgengine/wgcfg/config.go

```diff
@@ -25,6 +25,7 @@ type Config struct {
 	MTU        uint16
 	DNS        []netaddr.IP
 	Peers      []Peer
+	Debug      *tailcfg.Debug
```

Comment:
How about just a bool instead? But seems a little weird putting this on wgcfg.Config.

Reply:
PTAL